source/refunding_payments/index.html.md.erb

```diff
@@ -5,231 +5,106 @@ weight: 75
 
 # Refunding payments
 
-GOV.UK Pay supports refunding payments. You can choose to refund part of a
-payment, or the full amount.
+You can refund all or part of a payment using the GOV.UK Pay API.
 
-Each payment has a refund status:
+You can also refund a payment by signing into the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login).
 
-<div style="height:1px;font-size:1px;">&nbsp;</div>
+Your payment service provider (PSP) will send refunds back to the source account of the original payment. You cannot refund a payment to another card or bank account.
```

Comment:
Referring to 'your PSP' might be confusing for services using Stripe?

Reply:
This phrasing was already in the live docs (https://docs.payments.service.gov.uk/refunding_payments/#what-happens-next) - but happy to tweak it if it's not working? Maybe "**The** payment service provider" might work better?

Reply:
Maybe we don't need to reference the PSP at all? 
`Refunds will be sent back to the source account of the original payment`?

Reply:
sounds good